bundles/org.openhab.binding.volvooncall/README.md

```diff
@@ -99,13 +99,14 @@ Multiple actions are supported by this binding. In classic rules these are acces
 Example
 
 ```
- val actions = getActions("volvooncall","volvooncall:vehicle:bridgeId:thingId")
+ val actions = getActions("volvooncall","volvooncall:vehicle:myVinNumber")
```

Comment:
I do not understand how this can be wrong, as `getActions` is based on your version of the binding, and is the only command possible to use to trigger/send commands from script/rules (tested by far, i.e. proposed `actions.get` does not generate anything).

My only modification is the input String in `getActions`, which was needed to get commands through.

Reply:
It's not wrong. It's technically possible to create a Thing with that UID. However, the official convention is to use `bridgeId:thingId`. But in this example it would probably be better to use the id of the other examples: `volvooncall:vehicle:glh:XC60`.

Reply:
Well, the issue is that `bridgeid:thingId` i.e `glh:XC60` as referred to in the initial binding does not work with the commands/actions. I.e. getActions does not generate anything, and the commands will not be executed.

What does work is `volvooncall:vehicle:"VIN of the vehicle associated with this Thing"`

Reply:
Meaning, `getActions("volvooncall", "volvooncall:vehicle:"VIN of the vehicle associated with this Thing").sendCommand ()` is the only setup in script/rule which works for me.

Reply:
Also supported by other user testing same syntax, ref
https://community.openhab.org/t/solved-help-with-sending-command/7046/196

The command send is acknowledged and confirmed by the binding (although command is not executed correctly, something which this PR aim to fix with the {} attribute included amongst others.)

Reply:
`volvooncall:vehicle:"VIN of the vehicle associated with this Thing"` only works because that is the definition of your thing and in your thing the `vin` value is set. The fact it works has nothing to do with your vin number. If you would set your thing to any other random name and use that name in the action it would still work. But depending on how you define a thing it can be either `volvooncall:vehicle:bridgeId:thingId` or `volvooncall:vehicle:thingId`. But the first is the actual preferred configuration.

Reply:
Realised that as well. 

Although, in my case, the Thing, i.e. vehicle, was generated using autodiscovery, and then automatically became `volvooncall:vehicle:thingId` where thingID = VIN.

May need to have both alternatives presented in README, i.e. autodiscovered vs scriptbased Thing?

Reply:
Corrected with new commit.